charts/core/src/Series/index.ts

```diff
@@ -1,3 +1,3 @@
-export { Bar, type BarProps } from './Bar';
+export { Bar, type BarHoverBehavior, type BarProps } from './Bar';
```

Comment:
this should fix build

```suggestion
export { Bar, BarHoverBehavior, type BarProps } from './Bar';
```